api/src/main/resources/workbench.yaml

```diff
@@ -724,7 +728,54 @@ paths:
           type: integer
           format: int64
           required: true
-          description: specifies version of cdr
+          description: specifies which cdr version
+      responses:
+        "200":
+          description: A cohortReviewId, creationTime and cohortCount
+          schema:
+            $ref: "#/definitions/CohortReviewInfo"
+
+  /api/v1/cohortreview/create/{cohortReviewId}:
+    get:
+      tags:
+        - cohortReview
+      description:  >
+        This endpoint will create an cohort review which is a participant cohort sample
+        specified by the review size parameter.
+      operationId: "createCohortReview"
+      parameters:
+        - in: path
+          name: cohortReviewId
+          type: integer
+          format: int64
+          required: true
+          description: specifies which cohort review
+        - in: query
+          name: size
+          type: integer
+          required: false
+          description: size of the participant cohort statuses sample
+      responses:
+        "200":
+          description: A cohortReviewId and cohort count
+          schema:
+            $ref: "#/definitions/CohortReviewInfo"
+
+  /api/v1/cohortreview/participants/{cohortReviewId}:
+    get:
+      tags:
+         - cohortReview
+      description:  >
+        Returns a collection of participants that for the specified cohortReviewId. This endpoint
+        does pagination based on page, limit, order and column.
+      operationId: "getParticipantCohortStatuses"
+      parameters:
+        - in: path
+          name: cohortReviewId
+          type: integer
+          format: int64
+          required: true
+          description: specifies which cohort review
         - in: query
           name: page
```

Comment:
FWIW, here's how I handled pagination for the materialize cohort endpoint:

https://github.com/all-of-us/workbench/blob/master/api/src/main/resources/workbench.yaml#L1049

There's a pageToken and pageSize parameter... the pageToken can encode the position in pagination (e.g. a participant ID and status) as well as the order, etc.; or it can have just the position in pagination and a hash of other parameters (which means we validate that everything else is the same.)

We don't have to be consistent here -- just wanted to make sure you knew what I was doing elsewhere.